AllJoyn/Platform/Bridge.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
```

Comment:
Not required. I shall fix that

---
In reply to: [75173869](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75173869) [](ancestors = 75173869)